.github/workflows/Tests.yml

```diff
@@ -18,8 +18,6 @@ jobs:
       matrix:
         version:
           - '1.3'
```

Comment:
Given that 1.6 is now the LTSR, how about testing 1.6, 1, nightly?  (and leave 1.3 behind?)

Reply:
Sure, that would be a separate PR though. I messed up with git here; the first two commits are already on master. I'll try again on top of #102

Reply:
Agreed, that would be separate PR.  It was just a note for the future...